_tags

```diff
@@ -3,6 +3,7 @@ true: warn(@5@8@10@11@12@14@23@24@26@29@40), bin_annot, safe_string, debug, cppo
 
 "src": include
 <src/*.{ml,mli,byte,native}>: package(dynlink), package(compiler-libs.common), package(ppx_tools.metaquot), package(result)
+<src/ppx_deriving_main.{ml,mli,byte,native}>: package(findlib.dynload), predicate(ppx_driver)
```

Comment:
`findlib.dynload` is a new feature, right? I think this needs an opam dependency bump.